packages/sources/us-census/README.md

```diff
@@ -0,0 +1,93 @@
+# Chainlink External Adapter for the US Census (Decennial and American Community Survey)
+
+The US Census adapter fetches the latest census results published by the United States Census Bureau and transforms the variables into user-friendly responses.
+
+Please be careful with the latitude/longitude and passed into this, since there is only validation done on existence of the lat/lng (they must be a valid value in the US).
+
+### Environment Variables
+
+| Required? |  Name   |                               Description                               | Options | Defaults to |
+| :-------: | :-----: | :---------------------------------------------------------------------: | :-----: | :---------: |
+|    ✅     | API_KEY | [US Census API key](https://api.census.gov/data/key_signup.html) to use |         |             |
+
+---
+
+### Input Parameters
+
+| Required? |   Name    |                                              Description                                              |                                        Options                                         | Defaults to |
+| :-------: | :-------: | :---------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------: | :---------: |
+|    ✅     | endpoint  |        The [Census Source/year](https://www.census.gov/data/developers/data-sets.html) to use         | dec_2010, acs5_2013, acs5_2014, acs5_2015, acs5_2016, acs5_2017, acs5_2018, acs5_2019  |             |
```

Comment:
While this makes sense in this context alone, I think this is slightly confusing with what we normally use `endpoint` for in our EA monorepo.

I'd recommend making `endpoint` optional and default to `census` (similar to the format we follow with the other EAs). Then what's currently used as `endpoint` here should rather be called `dataset` or something like that.

Does that make sense to you?

Reply:
Yeah, that makes a lot of sense. I went back and forth on this, looking at other impls. I'll include that in my changes!